src/components/EstimatedFee.tsx

```diff
@@ -49,9 +49,9 @@ export const EstimatedFee = (): React.ReactElement => {
   return (
     <div>
       <Typography variant="body1" color="secondary">
-        Estimated source Fee:
+        {`Estimated source Fee ${estimatedFeeLoading ? 'Loading' : ''}`}
         <span className={`${estimatedFeeLoading ? classes.blurred : ''} ${classes.amount}`}>{amount}</span>
-        {chainTokens}
+        {!estimatedFeeLoading && chainTokens}
```

Comment:
If we switch to `...` I'd also always display tokens here (unconditionally).

Reply:
What do you mean?  not sure i get this comment.

Reply:
I think Tomasz means that if we replace `Loading` with `...` There is no need to have a condition on wheather the chainTokens should appear or not (and IMO he is right)